packages/slice-machine/pages/changes.tsx

```diff
@@ -13,16 +13,19 @@ import {
 } from "@components/ChangesEmptyPage";
 import { Button } from "@components/Button";
 import { AuthStatus } from "@src/modules/userContext/types";
-import { useUnSyncChanges } from "@src/hooks/useUnSyncChanges";
+import { unSyncStatuses, useUnSyncChanges } from "@src/hooks/useUnSyncChanges";
 import { isLoading } from "@src/modules/loading";
 import { LoadingKeysEnum } from "@src/modules/loading/types";
 import useSliceMachineActions from "@src/modules/useSliceMachineActions";
-import {
-  SoftDeleteDocumentsDrawer,
-  HardDeleteDocumentsDrawer,
-  ReferencesErrorDrawer,
-} from "@components/DeleteDocumentsDrawer";
+// import {
```

Comment:
Are these coming back with the `delete` feature?

Reply:
Yep!